### DISCUSSION TITLE
unregistered changes in changelog for ps v4 to ps v5

### DISCUSSION THREAD

**Discussion by mercerraa** [INSPIRE-MIF/helpdesk#192] [General]
I am very new to Inspire and so apoligise if my question is trivial or misplaced.

In [version 4 of the ps schema](https://inspire.ec.europa.eu/schemas/ps/4.0/ProtectedSites.xsd) siteDesignation, legalFoundationDate and legalFoundationDocument are siblings, which seems wrong given the UML description of Protected Sites. [Version 5](https://inspire.ec.europa.eu/schemas/ps/5.0/ProtectedSites.xsd) has changed this but I cannot find a reference to this change in the [change log](https://github.com/INSPIRE-MIF/application-schemas/releases/tag/2024.1). 
Have I misunderstood the change log, the schema or both?

**Comment by sMorrone** [comment]
Dear @mercerraa 
the mentioned [changelog](https://github.com/INSPIRE-MIF/application-schemas/releases/tag/2024.1) briefly summarizes the changes in the Protected Sites  (**[ps] - v5.0**) schema and links to the relevant issues in the issue tracker of the [application-schemas](https://github.com/INSPIRE-MIF/application-schemas) repository (below is the screenshot of related rows):
![image](https://github.com/INSPIRE-MIF/helpdesk/assets/13329248/3c786851-ca74-48ca-ad4b-ea0569892c5a)
So, 
- the changes to the PS data model (and consequently to the associated schema) originate from the Amendment of the 1089 /2010 (the INSPIRE Implementing Rules). 
- Details of the schema changes can be found in issues [#13](https://github.com/INSPIRE-MIF/application-schemas/issues/13) and [#14](https://github.com/INSPIRE-MIF/application-schemas/issues/14) of the application-schemas repository. 

Similarly, you will find mention of the changes in the PS Technical Guidelines in the changelog of the [2024.1 TG release](https://github.com/INSPIRE-MIF/technical-guidelines/releases/tag/v2024.1) in turn linking for more details the issue [#129](https://github.com/INSPIRE-MIF/technical-guidelines/issues/129) in the [technical-guidelines](https://github.com/INSPIRE-MIF/technical-guidelines) repository.
The HTML view of the INSPIRE UML models always refers to the latest version of the data model /schema.

**Comment by mercerraa** [comment]
Thank you. Wow, this is hard to navigate. Evidently I was looking at the
wrong change log and looking in the wrong discussion.



Den tors 18 apr. 2024 kl 16:20 skrev Stefania Morrone <
***@***.***>:
